### DISCUSSION TITLE
How to connect to nextjs site?

### DISCUSSION THREAD

**Discussion by OldPranoy** [BlackThunder01001/winerp#8] [Q&A]
I want to connect my discord.py bot to my nextjs site and i wanna call some event from nextjs site to dpy bot how can I do it with this lib?

**Comment by shahriyardx** [comment]
You can make a webserver using quart or any other async framework of your choice and have different routes for covering all your winerp client routes of your bot

Then you can simply make Request to the webserver from NextJS using fetch

I do this currently. 

Bus since winerp has a websocket server you can connect to it using node's websockets package and make direct request without the intermediary webserver if you like

But making webserver is more convenient.
